### ISSUE TITLE
Anti-Mi2 antibody positivity

### ISSUE THREAD

**Issue by eswietlik** [obophenotype/human-phenotype-ontology#6603]
**Preferred term label:** Anti-Mi2 antibody positivity

**Synonyms**
Anti-Mi-2/NuRD antibody positivity

**Definition (free text, please give PubMed ID)**
The Mi-2 antigen is a nuclear antigen, component of the Nucleosome Remodelling-Deacetylase (NuRD) complex involved in transcription regulation. Autoantibodies targeting the Mi-2 nuclear antigen are a serologic hallmark of idiopathic inflammatory myopathies, with a diagnostic sensitivity and specificity of approximately 4–18% and 98–100%, respectively. They are most strongly associated with dermatomyositis and correlate with HLA-DR7. PMID: 30938432

**Parent term (use hpo.jax.org/app)**
Antinuclear antibody positivity HP:0003493

**Diseases characterized by this term? (e.g. Orphanet or OMIM number)**
Dermatomyositis ORPHA:221

**Your nano-attribution (ORCID)**
https://orcid.org/0000-0002-4095-8489

**Comment by pnrobinson** [comment]
added
